### PR TITLE
Fix documentation typos and mistakes

### PR DESCRIPTION
**Pull request by gucong3000**
> Which issue, if any, is this issue related to?

None, as it's a documentation fix.

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

**Comment by hudochenkov** [comment]
I see you're finding many typos while working on translations. Thank you for fixing them!

Let's keep this PR open, and you will add all typo fixes here. This way you won't need to create a lot of PRs, and we don't need to click a lot to approve them and merge. What do you think?

File: lib/rules/function-url-quotes/README.md [hunk 1]
```diff
@@ -92,11 +92,11 @@ a { background: url(x.jpg); }
 
 ### `except: ["empty"]`
 
-Reverse the primary option if the function has no arguments.
+Reverse the primary option for functions that has no arguments.
```
Comment (jeddy3) [inline lib/rules/function-url-quotes/README.md]:
```suggestion
Reverse the primary option for functions that have no arguments.
```

File: lib/rules/media-query-list-comma-space-after/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Require a single space or disallow whitespace after the commas of media query li
 ```css
 @media screen and (color), projection and (color) {}
 /**                      ↑
- *            These commas */
+ * The space after This comma */
```
Comment (jeddy3) [inline lib/rules/media-query-list-comma-space-after/README.md]:
```suggestion
 * The space after this comma */
```

File: lib/rules/property-blacklist/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Specify a blacklist of disallowed properties.
 ```css
 a { text-rendering: optimizeLegibility; }
 /** ↑
- * These properties */
+ * This propertie */
```
Comment (jeddy3) [inline lib/rules/property-blacklist/README.md]:
```suggestion
 * This property */
```

File: lib/rules/property-case/README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ Specify lowercase or uppercase for properties.
 ```css
     a { width: 1px; }
 /**     ↑
- * These properties */
+ * This propertie */
```
Comment (jeddy3) [inline lib/rules/property-case/README.md]:
```suggestion
 * This property */
```

File: lib/rules/property-no-unknown/README.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ Disallow unknown properties.
 ```css
 a { heigth: 100%; }
 /** ↑
- * These properties */
+ * This propertie */
```
Comment (jeddy3) [inline lib/rules/property-no-unknown/README.md]:
```suggestion
 * This property */
```

File: lib/rules/property-no-vendor-prefix/README.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ Disallow vendor prefixes for properties.
 ```css
 a { -webkit-transform: scale(1); }
 /**  ↑
- * These prefixes */
+ * This prefixe */
```
Comment (jeddy3) [inline lib/rules/property-no-vendor-prefix/README.md]:
```suggestion
 * This prefix */
```

File: lib/rules/property-whitelist/README.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ Specify a whitelist of allowed properties.
 ```css
 a { display: block; }
 /** ↑
- * These properties */
+ * This propertie */
```
Comment (jeddy3) [inline lib/rules/property-whitelist/README.md]:
```suggestion
 * This property */
```

File: lib/rules/selector-attribute-quotes/README.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ Require or disallow quotes for attribute values.
 ```css
 [target="_blank"] {}
 /**     ↑      ↑
- * These quotes */
+ * This quote */
```
Comment (jeddy3) [inline lib/rules/selector-attribute-quotes/README.md]:
```suggestion
 * These quotes */
```

File: lib/rules/selector-max-specificity/README.md [hunk 9]
```diff
@@ -12,7 +12,7 @@ Visit the [Specificity Calculator](https://specificity.keegan.st) for visual rep
 
 This rule ignores selectors with variable interpolation (`#{$var}`, `@{var}`, `$(var)`).
 
-This rule resolves nested selectors before calculating the specificity of a selector.
+This rule resolves nested selectors before counting the the specificity of a selector. Each selector in a [selector list](https://www.w3.org/TR/selectors4/#selector-list) is evaluated separately.
```
Comment (jeddy3) [inline lib/rules/selector-max-specificity/README.md]:
```suggestion
This rule resolves nested selectors before counting the specificity of a selector. Each selector in a [selector list](https://www.w3.org/TR/selectors4/#selector-list) is evaluated separately.
```

File: lib/rules/selector-max-type/README.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # selector-max-type
 
-Limit the number of type selectors in a selector.
+Limit the number of type in a selector.
```
Comment (jeddy3) [inline lib/rules/selector-max-type/README.md]:
```suggestion
Limit the number of type selectors in a selector.
```

File: lib/rules/selector-no-vendor-prefix/README.md [hunk 11]
```diff
@@ -5,7 +5,7 @@ Disallow vendor prefixes for selectors.
 ```css
 input::-moz-placeholder {}
 /**     ↑
- * These prefixes */
+ * This prefixe */
```
Comment (jeddy3) [inline lib/rules/selector-no-vendor-prefix/README.md]:
```suggestion
 * This prefix */
```

**Pull request merged**